components/script/dom/node.rs

```diff
@@ -1203,8 +1207,7 @@ impl Node {
                 match last_child.and_then(|node| {
                     node.inclusively_preceding_siblings()
                         .filter_map(DomRoot::downcast::<Element>)
-                        .filter(|elem| is_delete_type(elem))
-                        .next()
+                        .find(|elem| is_delete_type(elem))
```

Comment:
I thought this would work but it doesn't compile. Please make sure that your patch compiles even if you are only doing requested changes, reviewers can be wrong.